example/lib/develop/model/ble_peripheral.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:equatable/equatable.dart';
+
+class BlePeripheral extends Equatable {
+  String name;
+  String id;
+  int rssi;
+  bool isConnected;
+
+  String get rssiString => '${rssi ?? '-'}dbm';
```

Comment:
`'${rssi ?? '-'}dbm'` is part of of the view layer so it should not be placed in the model

Reply:
Fixed.